coldfront/plugins/qumulo/views/update_allocation_view.py

```diff
@@ -11,21 +14,30 @@
     AllocationAttributeChangeRequest,
     AllocationChangeRequest,
     AllocationChangeStatusChoice,
+    AllocationLinkage,
     AllocationUser,
 )
+from coldfront.core.user.models import User
+from coldfront.core.utils.mail import send_email
```

Comment:
Request a new line between coldfront.core imports and coldfront plugin qumulo imports